src/dialogs/examples/teams/NotifyDialog.ts

```diff
@@ -0,0 +1,46 @@
+import * as builder from "botbuilder";
+import { TriggerActionDialog } from "../../../utils/TriggerActionDialog";
+import { DialogIds } from "../../../utils/DialogIds";
+import { DialogMatches } from "../../../utils/DialogMatches";
+import { Strings } from "../../../locale/locale";
+import * as teams from "botbuilder-teams";
+
+export class NotifyDialog extends TriggerActionDialog {
+    private static async step1(session: builder.Session, args?: any | builder.IDialogResult<any>, next?: (args?: builder.IDialogResult<any>) => void): Promise<void> {
+        if (session.message.source === "msteams")
+        {
+            // Actvity notications will only work in 1 to 1 chat
+            if (session.message.sourceEvent.channel == null)
```

Comment:
Let's be consistent and use isMessageFromChannel

Reply:
Done.